cmd/testpeers.go

```diff
@@ -38,19 +64,30 @@ func newTestPeersCmd(runFunc func(context.Context, io.Writer, testPeersConfig) e
 
 	bindTestFlags(cmd, &config.testConfig)
 	bindTestPeersFlags(cmd, &config)
+	bindP2PFlags(cmd, &config.P2P)
+	bindDataDirFlag(cmd.Flags(), &config.DataDir)
+	bindTestLogFlags(cmd.Flags(), &config.Log)
 
 	return cmd
 }
 
 func bindTestPeersFlags(cmd *cobra.Command, config *testPeersConfig) {
 	const enrs = "enrs"
-	cmd.Flags().StringSliceVar(&config.ENRs, "enrs", nil, "[REQUIRED] Comma-separated list of each peer ENR address.")
-	cmd.Flags().StringSliceVar(&config.P2PRelays, "p2p-relays", []string{"https://0.relay.obol.tech,https://2.relay.obol.tech"}, "Comma-separated list of each peer P2P relay address.")
+	cmd.Flags().StringSliceVar(&config.ENRs, enrs, nil, "[REQUIRED] Comma-separated list of each peer ENR address.")
+	cmd.Flags().DurationVar(&config.KeepAlive, "keep-alive", 30*time.Minute, "Time to keep TCP node alive after test completion, so connection is open for other peers to test on their end.")
+	cmd.Flags().DurationVar(&config.LoadTestDuration, "load-test-duration", 30*time.Second, "Time to keep running the load tests in seconds. For each second a new continuous ping instance is spawned.")
```

Comment:
Previously you implemented great [duration](https://github.com/ObolNetwork/charon/blob/main/cmd/duration.go) structure, why not to leverage it here to let user set arbitrary duration, not just seconds?

Reply:
The cobra CLI works in a similar way as my duration structure, meaning the input can be `--load-test-duration=5s` and also `--load-test-duration=10m30s`. However, in my implementation I am starting a new continuous ping each second, so `--load-test-duration=10ms` does not make much sense for load testing. I can probably change the description to:
```suggestion
	cmd.Flags().DurationVar(&config.LoadTestDuration, "load-test-duration", 30*time.Second, "Time to keep running the load test. For each second a new continuous ping instance is spawned.")
```